packages/core/admin/admin/src/components/GuidedTour/Stepper/Modal/StepperModal.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { useIntl } from 'react-intl';
+import { pxToRem } from '@strapi/helper-plugin';
+import { Typography } from '@strapi/design-system/Typography';
+import { Button } from '@strapi/design-system/Button';
+import { Box } from '@strapi/design-system/Box';
+import { Flex } from '@strapi/design-system/Flex';
+import Content from '../../GuidedTourModal/Content';
+import StepLine from '../StepLine';
+import StepNumberWithPadding from './StepNumberWithPadding';
+
+const LeftSideBox = styled(Flex)`
+  margin-right: ${({ theme }) => theme.spaces[8]};
+  flex-shrink: 0;
```

Comment:
Why do you need to set the shrink here ?

Reply:
Because even though we said to this box to be 30px width, it shrinks when there's lot of content on their right side. But actually I can replace the shrink 0 by using minWidth 30px instead, WDYT?

![image](https://user-images.githubusercontent.com/71838159/151163826-a1721ade-2b52-4afc-a42f-4335e1db2719.png)